sources/packages/backend/apps/db-migrations/src/sql/NotificationMessages/Insert-offering-change-request-inprogress-with-student-message.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO
+  sims.notification_messages(id, description, template_id)
+VALUES
+  (
+    13,
+    'Offering Change Request Inprogress With Student',
```

Comment:
shouldn't we also include  `application`, so it is `Application Offering Change Request In progress With Student',